spot_wrapper/spot_dance.py

```diff
@@ -0,0 +1,82 @@
+import time
+
+from bosdyn.choreography.client.choreography import (
+    load_choreography_sequence_from_txt_file,
+)
+from bosdyn.client import ResponseError
+from bosdyn.client.exceptions import UnauthenticatedError
+from bosdyn.client.robot import Robot
+from bosdyn.choreography.client.choreography import ChoreographyClient
+from bosdyn.client.lease import LeaseClient
+
+
+class SpotDance:
+    def __init__(
+        self,
+        robot: Robot,
+        choreography_client: ChoreographyClient,
+        lease_client: LeaseClient,
```

Comment:
Don't need the lease_client parameter anymore and then I think you are good to go (following a hardware test)